concepts/objects/about.md

```diff
@@ -0,0 +1,251 @@
+# About
+
+## Explanation
+
+Besides primitive data types like `number` and `string`, there is another important data type in JavaScript called `object`.
+Objects are collections of key-value pairs.
+In other languages they are referred to as maps or dictionaries and often times values need to have the same data type.
```

Comment:
```suggestion
In other languages they are referred to as maps or dictionaries and often times all values need to have the same data type.
```

Reply:
My initial guess would have been that they're more similar to structs or data classes in other languages, which can have varying value types. Perhaps you could briefly mention how those two relate?

Reply:
Also, on this line "often times" isn't familiar sounding English to me. Just "often" without the "times" is clearer 🙂

Reply:
I think it's a US phrase mostly

Reply:
@SaschaMann you are totally right with that first comment, in my head it was clear that we will cover that struct/class aspect of objects elsewhere but I forgot to also make this clear for the reader. Rewrote that section. Removed the "times" as well.